lib/src/logging_service.dart

```diff
@@ -54,6 +58,31 @@ class LoggingService {
     enable ? _enabledChannels.add(channel) : _enabledChannels.remove(channel);
   }
 
+  /// Called to register service extensions.
+  ///
+  /// Note: ideally this will be replaced w/ inline registrations within the
+  /// flutter foundation binding (see e.g., https://github.com/flutter/flutter/pull/21505).
+  void initServiceExtensions() {
+    _registerServiceExtension(
+        name: 'logging',
```

Comment:
perhaps match flutter conventions and namespace service extensions. 
E.g. 
perhaps these extensions should be
'flutter.logs.enable'
and 
flutter.logs.channels